recipes/nipype/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "0.12.0" %}
+
+package:
+  name: nipype
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/nipy/nipype/archive/{{ version }}.tar.gz
+  md5: 4d010bae6028cf4622bc1946c679fabc
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - numpy>=1.6.2
+    - scipy>=0.11
+    - networkx>=1.7
+    - traits>=4.3
+    - python-dateutil>=1.5
+    - nibabel>=2.0.1
+    - nose>=1.2
+    - future==0.15.2
+    - simplejson>=3.8.0
+    - prov>=1.4.0
+    - xvfbwrapper
+    - psutil
+    - funcsigs
+
+  run:
+    - numpy>=1.6.2
+    - scipy>=0.11
+    - networkx>=1.7
+    - traits>=4.3
+    - python-dateutil>=1.5
+    - nibabel>=2.0.1
+    - nose>=1.2
+    - future==0.15.2
```

Comment:
Not trying to say it is wrong. Maybe there is a reason. Just trying to understand it. That's all.